### PR TITLE
app-crypt/sbsigntool: Bump to version 0.9.1

### PR DESCRIPTION
**Pull request by ConiKost**
Bump to version 0.9.1

Closes: https://bugs.gentoo.org/647914
Closes: https://bugs.gentoo.org/652094

### CODE REVIEW

File: app-crypt/sbsigntool/sbsigntool-0.9.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN}s"
+
+inherit autotools eutils toolchain-funcs
```
Comment (Zlogene) [inline app-crypt/sbsigntool/sbsigntool-0.9.1.ebuild]: do you need `eutils` here?

File: app-crypt/sbsigntool/sbsigntool-0.9.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		ia64)  iarch=ia64 ;;
+		x86)   iarch=ia32 ;;
+		*)     die "unsupported architecture: ${ARCH}" ;;
+	esac
+	sed -i "/^EFI_ARCH=/s:=.*:=${iarch}:" configure.ac || die
+	sed -i 's/-m64$/& -march=x86-64/' tests/Makefile.am || die
+	sed -i "/^AR /s:=.*:= $(tc-getAR):" lib/ccan/Makefile.in || die #481480
+
+	# Apply OpenSSL 1.1 fix
+	epatch "${FILESDIR}"/${MY_P}-openssl-1.1.0-compat.patch
```
Comment (Zlogene) [inline app-crypt/sbsigntool/sbsigntool-0.9.1.ebuild]: `eapply`

File: app-crypt/sbsigntool/sbsigntool-0.9.1.ebuild (hunk 1 above)
Comment (ConiKost) [reply app-crypt/sbsigntool/sbsigntool-0.9.1.ebuild]: Thanks! Pushed a new one.

File: app-crypt/sbsigntool/sbsigntool-0.9.1.ebuild (hunk 2 above)
Comment (ConiKost) [reply app-crypt/sbsigntool/sbsigntool-0.9.1.ebuild]: Thanks! Pushed a new one.

**Comment by ConiKost** [comment]
Pushed a new one to solve the CI error

**Pull request closed without merging**